apps/webapp/src/fetchers/balances.ts

```diff
@@ -35,29 +32,19 @@ type NormalizedBalance = AssetBalance & {
   account: { index: number; address: string };
 };
 
-// Given an asset has many denom units, the amount should be formatted using
-// the exponent of the display denom (e.g. 1,954,000,000 upenumbra = 1,954 penumbra)
-export const displayDenom = (res?: AssetsResponse): { display: string; exponent: number } => {
-  const display = res?.denomMetadata?.display;
-  if (!display) return { display: 'unknown', exponent: 0 };
-
-  const match = res.denomMetadata?.denomUnits.find(d => d.denom === display);
-  if (!match) return { display, exponent: 0 };
-
-  return { display, exponent: match.exponent };
-};
```

Comment:
Eliminated this helper function entirely, as it just takes an `AssetsResponse` (a thin wrapper around `DenomMetadata`) and returns just the display denom's exponent.

Going forward, we want to use the `ValueViewComponent` to display balances, which requires an entire `DenomMetadata` object. So we don't want to just strip a few properties out of that object here.